proposal/comp110_architecture_proposal.tex

```diff
@@ -6,72 +6,131 @@
-\item[Full text link:] Give the URL of a downloadable PDF of the paper, if you can find one
-\item[Comments:] Write a few sentences on how you found the article and why you believe it is relevant and/or important.
-\end{description}
-
-\section*{Paper 5}
-\begin{description}
-\item[Title:] Title of paper
-\item[Citation:] \cite{bibtex_key}
-\item[Abstract:] Copy and paste the abstract here
-\item[Web link:] Give the URL of the paper in IEEE Xplore, ACM Digital Library, or similar
-\item[Full text link:] Give the URL of a downloadable PDF of the paper, if you can find one
-\item[Comments:] Write a few sentences on how you found the article and why you believe it is relevant and/or important.
-\end{description}
-
-\bibliographystyle{ieeetr}
-\bibliography{comp110_architecture}
-
+	
+	\maketitle
+	
+	\section*{Topic}
+	
+	My essay will be on procedural level generation for a 2D platform game.
+	
+	
+	\section*{Paper 1}
+	% This is an example! Replace the details with a paper relevant to your chosen topic.
+	\begin{description}
+		\item[Title:] A Framework for Analysis of 2D Platformer Levels
+		\item[Citation:] \cite{tricky}
+		\item[Abstract:] ``Levels are the space where a player explores the rules and mechanics
+		of a game; as such, good level design is critical to the game design
+		process. While there are many broad design principles, level
+		design is inherently genre-specific due to the wide variety of rules
+		and types of challenge found between genres. Determining genrespecific
+		design principles requires an in-depth analysis of games
+		within the genre. We present such an analysis for the 2D platformer
+		genre, examining level components and structure with a view to
+		better understanding their level design. We then use this analysis
+		to present a model for platformer levels, specifically focusing on
+		areas of challenge. Our framework provides a common vocabulary
+		for these items and provides level designers with a method for
+		thinking about elements of platformers and how to compose them
+		to create interesting and challenging levels.''
+		\item[Web link:] \url{http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.494.4080&rep=rep1&type=pdf}
+		\item[Full text link:] \url{http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.494.4080&rep=rep1&type=pdf}
+		\item[Comments:] This is an informative article on the more ticker parts if the level design, I believe this to be relevant to the point as just finding techniques to finish the level does not contribute to the players enjoyment of the game and finding harder routes is a way of making the player enjoy it more. cited by over 40 people although it is not many it still shows that a few dozen people have. This was found through the internet and key word searches.
+	\end{description}
+	
+	\section*{Paper 2}
+	\begin{description}
+		\item[Title:] Empirical evaluation of procedural level
+		generators for 2D platform games
```

Comment:
This is a review paper. Not a primary research paper. Consider looking through some of the sources that this paper cites to find primary research.